src/routers/filing.py

```diff
@@ -78,3 +78,17 @@ async def patch_filing(request: Request, lei: str, period_name: str, field_name:
 @requires("authenticated")
 async def update_task_state(request: Request, lei: str, period_name: str, task_name: str, state: StateUpdateDTO):
     await repo.update_task_state(request.state.db_session, lei, period_name, task_name, state.state, request.user)
+
+
+@router.get("/institutions/{lei}/filings/{period_name}/contact_info", response_model=List[ContactInfoDTO])
+@requires("authenticated")
+async def get_contact_info(request: Request, lei: str, period_name: str):
+    return await repo.get_contact_info(request.state.db_session, lei, period_name)
+
+
+@router.post("/institutions/{lei}/filings/{period_name}/contact_info", response_model=ContactInfoDTO)
+async def post_contact_info(request: Request, lei: str, period_name: str, contact_info_obj: ContactInfoDTO = None):
```

Comment:
we can just call the parameter `contact_info`, no need to append `_obj` to the end; also we shouldn't have this default to None; this is a required payload.